src/backend/cdb/partitionselection.c

```diff
@@ -91,14 +91,12 @@ eval_part_qual(ExprContext *econtext, TupleTableSlot *inputTuple, List *qualList
  *		Input parameters:
  *		pn: parent PartitionNode
  *		accessMethods: PartitionAccessMethods
- *		root_oid: root table Oid
  *		value: partition key value
- *		exprTypid: type of the expression
```

Comment:
It seems, that exprTypid have to be returned to the annotation.

Reply:
Good catch! Thanks, fixed.